packages/core-contracts/contracts/token/ERC721.sol

```diff
@@ -234,14 +236,8 @@ contract ERC721 is IERC721, IERC721Metadata, ERC721Storage {
         if (AddressUtil.isContract(to)) {
             try IERC721Receiver(to).onERC721Received(msg.sender, from, tokenId, data) returns (bytes4 retval) {
                 return retval == IERC721Receiver.onERC721Received.selector;
```

Comment:
I believe this OZ implementation is wrong. The function is supposed to return false, not throw.

Reply:
I see your point (and in fact I don't agree at all with OZ's) but the reason they are doing it is to bubble up any revert from the underlying contract to better identify the issue.

So, not pro or against this change :)

Reply:
Let's keep this one. I am pro maintaining OZ code, until its arguably incorrect.